src/components/quote.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+
+const Quote = () => {
+  const apiUrl = 'https://api.api-ninjas.com/v1/quotes';
+  const category = 'life';
+  const [quoteText, setQuoteText] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      try {
+        const response = await fetch(`${apiUrl}?category=${category}`, { headers: { 'X-Api-Key': 'ol8rLaI2WXeFoWWBPY4sQA==yP7NsdMF8EyLCJWx' } });
+        if (!response.ok) {
+          throw new Error(`API request failed with status code ${response.status}.`);
+        }
+        setIsLoading(false);
+        const data = await response.json();
+        setQuoteText(data[0].quote);
+      } catch (error) {
+        setIsLoading(false);
```

Comment:
- [ ] Instead of duplicating `setIsLoading(false) you can put it after the catch block, it will run after everything inside the try-catch block is done with :+1:

Reply:
Hi @VICTOR-CHUKWU ,

Thanks for your time and for reviewing my code. I will consider your advice and update my code accordingly. 🙏